### DISCUSSION TITLE
Widget Rotation

### DISCUSSION THREAD

**Discussion by jwestes** [RudolphRiedel/FT800-FT813#72] [Q&A]
Obviously, bitmaps can be rotated.  Is it possible to rotate widgets to have a progress bar that goes right to left instead of left to right?

**Comment by RudolphRiedel** [comment]
Unfortunately this is not possible, my favourite in this regard is that the vertical slider goes from top to bottom.

Well, it eats computrons but instead of this:
CMD_SLIDER(100, 100, 15, 200, 0, sliderval, MAXSLIDER)

I am using it like this:
CMD_SLIDER(100, 100, 15, 200, 0, MAXSLIDER-sliderval, MAXSLIDER)

Or I rather put this into the touch function to only calculate it when necessary:
tracker_val = tracker_val / 536870; /* -> 8000 max */
speed = 8000 - tracker_val;

**Comment by jwestes** [comment]
I figured out that I can also use a scroll and adjust the size and value to make it look like a progress bar that goes from either end.  It's not perfect, because the fill area on the scrollbar is not quite as big as the fill area on the progress bar, but it works well.
